### PR TITLE
feat:adds the input component

### PR DESCRIPTION
**Pull request by FarhanMobashir**
input
- adds basic text input
- adds required style
- adds the input status message component
- add readonly variant
- adds the disbaled variant

### CODE REVIEW

File: src/components/input.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/* input */
+.label {
+  font-size: var(--tx-20);
+  font-weight: var(--tx-medium);
+  margin-top: 20px;
```
Comment (Hamdrive) [inline src/components/input.css:5]: Use 'rems', preferable CSS variables

**Review by Hamdrive — COMMENTED** [review]
Good work done, ready to push!

File: src/components/input.css (hunk 1 above)
Comment (FarhanMobashir) [reply src/components/input.css:5]: yup will correct it. thanks

File: src/pages/components.html [hunk 2]
```diff
@@ -442,7 +442,7 @@ <h3 class="price-text">$19.90</h3>
             />
             <label class="label" for="text">Email </label>
             <input type="text" class="text-input" pattern="/mobashir/" />
```
Comment (MaruthiPothuganti) [inline src/pages/components.html:444]: Suggestion: input type="email" can be used for email inputs

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended changes: change of type attribute of email input field

**Pull request merged**